docs/dev/contributing.md

```diff
@@ -37,16 +37,40 @@ Have you fixed a bug or written a new check and want to share it? Many thanks!
 In order to ease/speed up our review, here are some items you can check/improve
 when submitting your PR:
 
-  * have a proper commit history (we advise you to rebase if needed).
-  * write tests for the code you wrote.
-  * preferably make sure that all tests pass locally.
-  * summarize your PR with an explanatory title and a message describing your
-    changes, cross-referencing any related bugs/PRs.
-  * use [Reno](#reno) to create a releasenote.
-  * open your PR against the `main` branch.
-  * for PRs from contributors with write access to the repository (for community PRs, will be done by Datadog employees):
-    + set the relevant `team/` label
-    + add a milestone to your PR (by default, use the highest milestone version available, ex: `7.49.0`)
+<details open>
+<summary>Contributor Checklist</summary>
+
+- [ ] Have a proper commit history (we advise you to rebase if needed).
+
+- [ ] Write tests for the code you wrote.
+
+- [ ] Preferably make sure that all tests pass locally.
+
+- [ ] Summarize your PR with an explanatory title and a message describing your changes, cross-referencing any related bugs/PRs.
+
+- [ ] Use [Reno](#reno) to create a release note.
+
+- [ ] Open your PR against the `main` branch.
+
+- [ ] Provide adequate QA/testing plan information.
+</details>
+<br>
+
+<details>
+<summary open>Reviewer Checklist</summary>
+
+- [ ] The added code comes with tests.
+
+- [ ] The CI is green.
```

Comment:
```suggestion
- [ ] The CI is green, all tests are passing (required or not).
```
Not sure about this as eventually we might not have this distinction, but still :D

Reply:
Good point